src/relion/cryolo_relion_it/icebreaker_histogram.py

```diff
@@ -0,0 +1,52 @@
+import gemmi
+import json
+import matplotlib.pyplot as plt
+from pathlib import Path
+import plotly.express as px
+import pandas as pd
+
+
+def create_json_histogram(working_directory):
+    icebreaker_path = str(working_directory / "External/Icebreaker_group/")
+    json_hist_file_name = "ice_hist.json"
+    if Path(icebreaker_path + "/particles.star").is_file():
+        data = extract_ice_column(icebreaker_path + "/particles.star")
+    else:
+        return None
+    if data is None:
+        print("No data extracted.")
+    df = pd.DataFrame(data, columns=["Relative estimated ice thickness"])
+    fig = px.histogram(
+        df,
+        x="Relative estimated ice thickness",
+        title="Histogram of Icebreaker estimated ice thickness <br>Total number of particles = "
+        + str(len(data)),
+    )
+    fig.write_json(icebreaker_path + "/" + json_hist_file_name)
```

Comment:
```suggestion
    fig.write_json(icebreaker_path / json_hist_file_name)
```
If plotly can't handle path objects then can still wrap this in `os.fspath(...)`, but I would expect it can handle them.